app/routes/_index.tsx

```diff
@@ -1,22 +1,84 @@
-import { Card, CardGroup, CardHeader } from "@trussworks/react-uswds";
+import {
+    Breadcrumb,
+    BreadcrumbBar,
+    BreadcrumbLink,
+    Card,
+    CardGroup,
+    CardHeader,
+    Header,
+    Label,
+    Link,
+    Select,
+} from "@trussworks/react-uswds";
+import React from "react";
 import { mergeMeta } from "~/lib/merge-meta";
 
 export const meta = mergeMeta(({ parentTitle }) => [{ title: `Meeting Spaces • ${parentTitle}` }]);
 
 export default function Index() {
     return (
-        <CardGroup>
-            <Card>
-                <CardHeader>
-                    <h1 className="usa-card__heading">Meeting Spaces</h1>
-                    <p className="font-sans text-base-darker">
-                        Austin Public Library Meeting Spaces are
-                        <strong className="font-bold"> free of charge</strong> and ideal for
-                        discussion groups, panels, and lectures. Both paper and online reservation
-                        requests are timestamped and processed in the order they are received.
-                    </p>
-                </CardHeader>
-            </Card>
-        </CardGroup>
+        <div className="flex justify-center">
+            <CardGroup className="min-w-[30rem] max-w-[62rem]">
+                <Card>
+                    <Header>
+                        <div className="object-contain">
+                            <img
+                                alt="Public Meeting Room"
+                                className="rounded-t-md"
+                                loading="lazy"
+                                src="/meetingSpacesHeader.jpg"
+                            />
+                        </div>
+                        <BreadcrumbBar className="ml-2">
+                            <Breadcrumb className="relative left-0">
```

Comment:
It's probably worth coordinating with Mario to see if it would be helpful for you to create a component for this, if this is the way we'll always have to use the `<Breadcrumb>` component

Reply:
It would probably be best to create a component for a header now to use later down the line. We can just use it for everything inside the project after this PR is finished
.

Reply:
I created a custom `<Breadcrumbs />` component; I was thinking it'd be easiest for each route to include a `breadcrumbLinks` array that'll contain the necessary links for whatever page the user is on, since that'll change with each page. Let me know if that works. It's currently defined as this in `_index.tsx`: 

```
    const breadcrumbLinks = [
        { href: "#", text: "Home" },
        { href: "#", text: "Meeting Spaces" },
    ];
```

And you call it as: `<Breadcrumbs links={breadcrumbLinks} />`